examples/TEM_data/README.md

```diff
@@ -0,0 +1,381 @@
+understand how the alloy composition and heat treatment influences the
+microstructure and how the microstructure in turn influences the alloy
+properties.  Based such understanding, new alloys with improved or
+tailored properties can be developed.
+
+In this very simple example, we will look at a single Al-Mg-Si alloy
+at two conditions, after exposing it for two different heat treatments.
+
+The first heat treatment we denote DA (for direct aged), where the
+alloy was solution heat treated for 5 min at 530C and then quenched to
+room temperature (RT) and then immediately aged for 5 hours at 185C.
+
+![profile-DA](figs/profile-DA.svg)
+
+*Figure: Heat treatment profile for condition A.  Direct aging.*
+
+Two bright-field TEM images were acquired from a sample prepared after
+the aging (marked with A in the profile above).  The first is a
+low-magnification images showing a dense distribution of needle-shaped
+beta" precipitates.
+
+![Overview, direct aging](figs/BF_100-at-m5-and-2_001.png)
+
+*Figure: Low-magnification TEM bright-field image;
+[pm:TEM_BF_100-at-m5-and-2_001]*
+
+The second image zooms in to a smaller region.  The dark dots are
+beta" precipitates viewed along the needle-direction.
+
+![zoom, direct aging](figs/040.png)
+
+*Figure: Zoom-in of the above TEM bright-field image; [pm:TEM_040]*
+
+The second heat treatment we denote NA (for natural aged), where the
+alloy after solution heat treatment was kept at RT for 1 month before
+aging.  It was also pre-baked for 24h at 90C.
+
+![profile-NA](figs/profile-NA.svg)
+
+*Figure: Heat treatment profile for condition B.  Natural aging.*
+
+An atomic-resolution high angle annular dark-field (HAADF) TEM image
+was acquired from a sample prepared after aging (marked with B in the
+profile above).  It shows three beta" precipitates viewed along the
+needle direction.
+
+![pre-baked](figs/6c8cm_008.png)
+
+*Figure: High angle annular dark field TEM image; [pm:TEM_6c8cm_008]*
+
+By analysing a set of such TEM images as showed above, one can
+characterise the alloy microstructure in terms of precipitates types
+and size distributions.
+
+To relate the microstructure to mechanical properties of the alloy,
+Vickers hardness measurements was performed at each of the two
+conditions.
+
+
+Data resources
+--------------
+The data resources can be divided into two types, *data sources*
+with our initial knowledge or raw data obtained directly from
+experiments and *data sinks* that correspond to different derived
+representations of the primary data sources.
+
+All data resources are uniquely identified with an IRI, where we are using
+the prefix [pm:] as a common namespace.
+
+All data resources used in this example are documented in the
+[data/resources.yaml] file, which will be used to populate the
+knowledge base in this demo.
+
+**Data sources**
+- **Raw TEM images** stored in the Gatan DM3 format.  These images are large
+  and can be downloaded from https://folk.ntnu.no/friisj/temdata/.
```

Comment:
Don't we have anywhere at SINTEF were we can have such things?

Reply:
Not as far as know, but I wish we would have... maybe data.sintef.no finally will give us that?